pages/blog/tooling-update-1.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "AsyncAPI tooling update 1"
+date: 2022-04-13T18:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+  - Tooling Status
+cover: /img/posts/tooling-update.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Maintainer
+excerpt: 'A short update on what have happened since last update around tooling'
+featured: true
+---
+
+Another update on what happen in our tools that help developers develop EDAs and APIs! :fire: 
+
+The previous two updates can be found here: [January](https://gist.github.com/jonaslagoni/4a0117e1432936b89e05b6a509d94259) and [February](https://gist.github.com/jonaslagoni/185890cd7e84ed95c14b8f0556788b7b), however going forward I will add a new blog post for each update. If you have any remarks or if something is missing, you can [suggest changes in through the website repository](https://github.com/asyncapi/website/tree/master/pages/blog/tooling-update-1.md). 
+
+If you want to include updates from your personal tooling, the [next blog post can be found here](https://github.com/asyncapi/website/pull/662), just leave a comment with the updates you want included.
```

Comment:
```suggestion
If you want to include updates from your personal tooling, the [next blog post PR can be found here](https://github.com/asyncapi/website/pull/662), just leave a comment with the updates you want to be included.
```
It's confusing to say it's a blog post link when it's really a PR, so I wanted to fix that 😸